src/libraries/System.Net.WebSockets/src/System/Net/WebSockets/ManagedWebSocket.cs

```diff
@@ -1365,15 +1365,15 @@ private async ValueTask EnsureBufferContainsAsync(int minimumRequiredBytes, Canc
                     Debug.Assert(numRead >= 0, $"Expected non-negative bytes read, got {numRead}");
                     if (numRead <= 0)
                     {
-                        ThrowIfEOFUnexpected(throwOnPrematureClosure);
+                        ThrowOnEOFUnexpected();
                         break;
                     }
                     _receiveBufferCount += numRead;
                 }
             }
         }
 
-        private void ThrowIfEOFUnexpected(bool throwOnPrematureClosure)
+        private void ThrowOnEOFUnexpected()
```

Comment:
Nit:
ThrowOn suggests it might not throw... probably better now as just ThrowEofUnexpected. It code also be attributed now as DoesNotReturn.